server/server.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"context"
+	"log"
+	"net"
+
+	pb "github.com/BetterGR/staff-microservice/staff_protobuf"
+	"google.golang.org/grpc"
+)
+
+// server is used to implement pb.StaffServiceServer
+type staffServer struct {
+	pb.UnimplementedStaffServiceServer
+}
+
+const (
+	port = ":50051"
+)
+
+/*// GetStudentGrade method
+func (s *gradesServer) GetStudentGrade(ctx context.Context, req *gpb.GradeRequest) (*gpb.GradeReply, error) {
+	log.Printf("Recevied", req.GetStudentId())
+	return &gpb.GradeReply{Grade: "100", Course: "test"}, nil
+}
+*/
+
+// GetStaffMember retrieves a specific staff member
```

Comment:
Code entities documentation (docs that are not inside functions) should be dotted.